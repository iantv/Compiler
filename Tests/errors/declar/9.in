struct a{
	struct a{
		int b;
	};
	int b;
};