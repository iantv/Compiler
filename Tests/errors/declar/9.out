error C3769: "a" : a nested class cannot have the same name as the immediately enclosing class in line 4 column 4
